site/en/user_guide.md

```diff
@@ -78,11 +83,13 @@ If you already have a METS file as indicated above, you can create a workspace
 and load the pictures to be processed with the following command: 
 
 ```sh
-ocrd workspace clone [URL of your mets.xml]
+ocrd workspace clone [URL of your mets.xml] [name of your workspace]
```

Comment:
Better:

```
ocrd workspace -d /path/to/workspace URL_OF_METS
```

(not the wording but that workspace directory should be set on `ocrd workspace` for consistency w/ other commands.